app/src/main/java/at/aau/serg/activities/GameScreenActivity.kt

```diff
@@ -334,6 +336,37 @@ class GameScreenActivity : AppCompatActivity() {
             allowedToPlayCard = true
             showToast(this, "You are now!")
         }
+
+        // highlight the current player card
+        val calcNextPlayerPosition = calculatePositionOfPlayer(nextPlayerIdx, myPlayerIndex, playerCount)
+        val calcCurrentPlayerPosition = calculatePositionOfPlayer(nextPlayerIdx-1, myPlayerIndex, playerCount)
+        // need to be on an extra thread with sleep!, because in the first round, the playerCardImageViews are not loaded yet
+        Thread {
+            Thread.sleep(1000) // because playerCardImageViews might not be loaded (null)
+            val nextPlayerImageView = findViewById<ImageView>(
+                resources.getIdentifier(
+                    "ivPlayer${calcNextPlayerPosition}Card",
+                    "id",
+                    packageName
+                )
+            )
+
+            var currentPlayerImageView: ImageView? = null
+            if(firstPlayedCard != null && winnerPlayerIndex != null && calculatePositionOfPlayer(
```

Comment:
This will probably also not work for `startSubRound` since `firstPlayedCard` and `winnerPlayerIndex` get set to null (so it will only work for the first surround and after the first card is played)

Reply:
This if checks whether the highlight of the previous player needs to be removed (is not the current player anymore). If there was no previous player, none needs to be replaced. The second part checks if the previous player would win the trick, otherwise we would remove the current winner highlight without this if

Reply:
Ok, it seems I made an error while debugging. Still, when the `isNextPlayer` function is called from `nextSubRound`, it doesn't mark the next player.

Reply:
Yes, i had also now the same problem. the problem was the same as before, that the image view was not initialized yet. 
I have fixed it now by adding a FragmentLifecycleCallback for FragmentResumed, this seems to fix the problem.